spec/models/user_group_spec.rb

```diff
@@ -1,22 +1,67 @@
 require 'rails_helper'
 
 RSpec.describe UserGroup, :type => :model do
-  context 'with groceries' do
+  describe 'group groceries' do
+    context 'with groceries' do
+      let(:user_group) { create(:user_group, :with_groceries) }
 
-    it 'scopes by unfinished' do
-      group = create(:user_group, :with_groceries)
-      result = group.active_groceries
+      it 'scopes by unfinished' do
+        result = user_group.active_groceries
+        expect(result).to eq user_group.groceries
+      end
 
-      expect(result).to eq group.groceries
+      it 'scopes by finished' do
+        user_group.groceries.update_all(finished_at: DateTime.now)
+        result = user_group.finished_groceries
+        expect(result).to eq user_group.groceries
+      end
+    end
+  end
+
+  describe 'privacy scoping' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.